tests/cases/base.php

```diff
@@ -871,8 +871,10 @@ protected function assertIdenticalInputProps($entity1, $entity2) {
             $this->assertIdentical($entity1->Type, $entity2->Type);
             $this->assertIdentical($entity1->Status, $entity2->Status);
             $this->assertIdentical($entity1->UserId, $entity2->UserId);
+        } elseif(is_a($entity1, '\MangoPay\PayInPaymentDetailsPaypal')) {
+            // TODO Add the right comparison
```

Comment:
We'll actually be adding a couple of new fields to this entity in the next week, so I suggest we leave it as is for now.